### PR TITLE
Move systemtests roles

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: .travis/setup-kubernetes.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ ENMASSE_DIR=${1}
 OPENSHIFT_CLIENT_URL=${2:-"https://github.com/openshift/origin/releases/download/v3.7.0/openshift-origin-client-tools-v3.7.0-7ed6862-linux-64bit.tar.gz"}
 MINIKUBE_RELEASE_URL=${3:-"https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64"}
 KUBECTL_RELEASE_URL=${4:-"https://storage.googleapis.com/kubernetes-release/release/v1.8.7/bin/linux/amd64/kubectl"}
-ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/environment.yml \
+ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/environment.yml \
```
Comment (lulf) [inline .travis/setup-kubernetes.sh]: It's not really ENMASSE_DIR anymore, so it should be renamed.

File: systemtests/scripts/test_func.sh [hunk 2]
```diff
@@ -29,7 +29,8 @@ function setup_test() {
     oc login -u ${OPENSHIFT_USER} -p ${OPENSHIFT_PASSWD} --insecure-skip-tls-verify=true ${OPENSHIFT_URL}
     oc adm --config ${KUBEADM} policy add-cluster-role-to-user cluster-admin $OPENSHIFT_USER
     export OPENSHIFT_TOKEN=`oc whoami -t`
-    ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/systemtests.yml --extra-vars "namespace=${OPENSHIFT_PROJECT} admin_user=${OPENSHIFT_USER}"
+    ansible-playbook ${CURDIR}/../ansible/playbooks/systemtests-dependencies.yml
+    ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/install.yml -i ${CURDIR}/../ansible/inventory/systemtests.inventory --extra-vars "namespace=${OPENSHIFT_PROJECT} admin_user=${OPENSHIFT_USER}"
```
Comment (lulf) [inline systemtests/scripts/test_func.sh]: Maybe I'm just confused, but I think ENMASSE_DIR will point to 'systemtests' in this case, and not 'templates/install', so the install playbook does not exist there.

Comment (kornys) [reply systemtests/scripts/test_func.sh]: @lulf I will rename ENMASSE_DIR you're right name is confusing, but install playbook is still there it is your install playbook but I run this playbook with systemtests.inventory

**Comment by kornys** [comment]
https://github.com/EnMasseProject/enmasse/issues/1342

**Comment by lulf** [comment]
@kornys I missed this in the review, but the HACKING.md should be updated with the correct commands and paths.

**Comment by kornys** [comment]
@lulf you're right, I will do that. Thanks!

**Pull request merged**